.github/workflows/test.yml

```diff
@@ -14,14 +14,28 @@ jobs:
     steps:
       - name: git checkout
         uses: actions/checkout@v3
+
       - name: Install Nix
         uses: DeterminateSystems/nix-installer-action@main
-      - uses: DeterminateSystems/magic-nix-cache-action@main
-      - uses: nicknovitski/nix-develop@v1
+
+      - name: Setup Nix cache
+        uses: DeterminateSystems/magic-nix-cache-action@main
+
+      - name: Setup Nix development environment
+        uses: nicknovitski/nix-develop@v1
         with:
           arguments: bsf/.#devShell
-      - name: run tests
+
+      - name: Install Task
```

Comment:
this is not needed either. Same reason. This is available from bsf develop shell

Reply:
Sorry didn't saw this `go-task@~3.37.2`